packages/server/Dockerfile

```diff
@@ -27,7 +27,7 @@ COPY scripts/integrations/oracle/ scripts/integrations/oracle/
 RUN /bin/bash -e scripts/integrations/oracle/instantclient/linux/x86-64/install.sh 
 
 COPY package.json .
-RUN yarn install --frozen-lockfile --production=true
+RUN yarn install --production=true
```

Comment:
I see that we don't copy the lock file across, how does that work?
Wouldn't it mean that the versions we have in our lock don't reflect what will be in the image?